apps/astarte_housekeeping_api/lib/astarte_housekeeping_api/realms/realms.ex

```diff
@@ -122,6 +138,7 @@ defmodule Astarte.Housekeeping.API.Realms do
 
   """
   def change_realm(%Realm{} = _realm) do
-    raise "TODO"
+    %Realm{}
+    |> Realm.changeset()
```

Comment:
Is there an actual API that leads to this? If not, my suggestion is to remove the whole code path